maat-court-data-api/src/main/java/gov/uk/courtdata/dces/controller/ConcorContributionsRestController.java

```diff
@@ -50,6 +50,19 @@ public ResponseEntity<List<ConcorContributionResponse>> concorContributionFiles(
         return ResponseEntity.ok(contributionResponses);
     }
 
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
+    @StandardApiResponse
+    @GetMapping(value = "/concor-contribution-file/{concorContributionId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Get the Contributions file for given concorContributionId")
```

Comment:
instead of 'contributions file', could this say 'Concor Contribution and associated XML' -- there is a separate entity called 'contribution-file' (see `ContributionFileController` and `ContributionFileService`). Also applies to the following `log.info(...)` line.

Reply:
Why not simply ConcorContribution - that's the name of the table?

Reply:
Sure. I was in two minds between this and keeping it consistent with the other endpoints. Making the change now.